src/amrex/extensions/MultiFab.py

```diff
@@ -1,13 +1,23 @@
 """
 This file is part of pyAMReX
 
-Copyright 2023 AMReX community
-Authors: Axel Huebl
+Copyright 2024 AMReX community
+Authors: Axel Huebl, David Grote
 License: BSD-3-Clause-LBNL
 """
 
+import numpy as np
+
 from .Iterator import next
 
+try:
```

Comment:
We need to put this into a function that has access to `amr`, so we only attempt an import if:
```py
if amr.Config.have_mpi:
```

Otherwise, this can cause issues in scenarios, e.g., running serially on Cray machines.